### ISSUE TITLE
styled svg renders viewbox attribute instead of viewBox

### ISSUE THREAD

**Issue by btakita** [solidjs/solid-styled-components#16]
```tsx
const Svg = styled('svg')<{ spin?:any, pulse?:any }>($p=>{
	return `
		${$p.spin ? `
			animation: ${spin} 2s 0s infinite linear;
		` : ''}
		${$p.pulse ? `
			animation: ${spin} 1s infinite steps(8);
		` : ''}
	`
})
<Svg viewBox="0 0 10 10"></Svg> # renders as <svg viewbox="0 0 10 10"></svg>
```

**Comment by ryansolid** [comment]
Hmm. Right, it isn't aware of SVG elements. Yeah in general I don't think this library supports SVG yet. I wonder if changing the implementation to use `<Dynamic>` will just handle this properly.

**Comment by ryansolid** [comment]
I believe #30 should add support for this. Released in 0.28.2.
